geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateAlterDestroyRegionCommands.java

```diff
@@ -907,6 +903,31 @@ private void validateRegionFunctionArgs(Cache cache, RegionFunctionArgs regionFu
                 new Object[] {regionFunctionArgs.getCompressor()}));
       }
     }
+
+    if (regionFunctionArgs.hasPartitionAttributes()) {
+      boolean partitionResolverFailure = false;
+      if (regionFunctionArgs.isPartitionResolverSet()) {
+        String partitionResolverClassName = regionFunctionArgs.getPartitionResolver();
+        Object partitionResolver = null;
+
+        try {
+          Class<?> compressorClass =
```

Comment:
Variable name is probably wrong(?) or copied from other code.

Since the class is of a known type, we should specify that here instead of using '?':

String partitionResolverClassName = PRResolver.class.getName();
Class<PartitionResolver> resolverClass = 
    (Class<PartitionResolver>)ClassPathLoader.getLatest().forName(partitionResolverClassName);
PartitionResolver partitionResolver = resolverClass.newInstance();